catalog/src/main/kotlin/com/adevinta/spark/catalog/model/Example.kt

```diff
@@ -24,12 +24,18 @@
 package com.adevinta.spark.catalog.model
 
 import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.RowScope
 import androidx.compose.runtime.Composable
 import com.adevinta.spark.components.snackbars.SnackbarHostState
 
 public data class Example(
     val name: String,
     val description: String,
     val sourceUrl: String,
-    val content: @Composable ColumnScope.(SnackbarHostState) -> Unit,
```

Comment:
I would prefer the previous api since the column it the most common layout that we have in the examples.
Instead of adding 2 nullable content that can be forgotten since now the content is optional having a mandatory content where we wrap our content in a Row like before doesn't add a huge overload on the complexity and let us have the better default almost every time.

Reply:
Thanks for the suggestion, I prefer my approach